.github/workflows/ci-pipeline.yml

```diff
@@ -53,3 +53,43 @@ jobs:
 
           docker tag $IMAGE_NAME $IMAGE_ID:$VERSION
           docker push $IMAGE_ID:$VERSION
+
+  Coverity-build:
+    runs-on: ubuntu-18.04
+    env:
+      IMAGE_DIR: ${{matrix.image}}
+      IMAGE_NAME: odp-ci-${{matrix.image}}
+    strategy:
+      fail-fast: false
+      matrix:
+        image: ['ubuntu_20.04-x86_64-coverity']
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build image
+        run: docker build --build-arg COVERITY_TOKEN=${{ secrets.COVERITY_TOKEN }} $IMAGE_DIR -t $IMAGE_NAME
+
+      - name: Log into registry
+        if: github.event_name == 'push'
+        run: echo "${{ secrets.CONTAINER_REGISTRY_TOKEN }}" | docker login ghcr.io -u ${{ secrets.CONTAINER_REGISTRY_USER }} --password-stdin
```

Comment:
On a quick glance the only difference compared to the current build job seems to be the Coverity token `--build-arg COVERITY_TOKEN=${{ secrets.COVERITY_TOKEN }}`. So could you just add the token to the current `Build image` step and add the image to the image array?

Reply:
That is true, it is the only difference. I was debating between these two options and I went with this one precisely because it highlights the difference between these images (and I didn't want to pass an unused argument).
Granted, this creates a lot of copy pasting for just that, so maybe it's not worth it. I'll squash the two together if you want.

Reply:
One more question - do we only want this one image for Coverity? I don't think there's much value in adding other Coverity images, but I still want to double-check, in case I'm missing something.

Reply:
> That is true, it is the only difference. I was debating between these two options and I went with this one precisely because it highlights the difference between these images (and I didn't want to pass an unused argument).
> Granted, this creates a lot of copy pasting for just that, so maybe it's not worth it. I'll squash the two together if you want.

I'd prefer easier code maintenance in the future, so please squash the code.


> One more question - do we only want this one image for Coverity? I don't think there's much value in adding other Coverity images (and jobs resulting from that), but I still want to double-check, in case I'm missing something.

A good point. We'd probably like to add Coverity support also for the ODP-DPDK repo later on. So perhaps it would  make sense to name this image for example `ubuntu_20.04-x86_64-coverity-linux-generic`.

Reply:
Ok, I'll squash the two and rename the image to `ubuntu_20.04-x86_64-coverity-linux-generic`.